bot/utils/redis_cache.py

```diff
@@ -131,6 +134,13 @@ def _from_typestring(key_or_value: Union[bytes, str], prefixes: _PrefixTuple) ->
         # Now we convert our unicode string back into the type it originally was.
         for prefix, _type in prefixes:
             if key_or_value.startswith(prefix):
+
+                # For booleans, we need special handling because bool("False") is True.
+                if prefix == "b|":
+                    value = key_or_value[len(prefix):]
+                    return bool(strtobool(value))
```

Comment:
Is it feasible to store it as 0/1 instead? Then it becomes `bool(int(value))`. I prefer that simplicity over dealing with actual strings and relying on a separate method.

Reply:
sure, but we're still turning that int into a string to store it in redis. But it might remove the reliance on strtobool, so I guess it'd be fine.